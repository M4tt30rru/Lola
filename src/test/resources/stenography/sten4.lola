##Import "std.lola"
##Find
 ##MethodDeclaration(_dec) = ##Any(_value);
 ##replace ##(_dec) {return ##(_value);}

T h(int x) = new T(x);
