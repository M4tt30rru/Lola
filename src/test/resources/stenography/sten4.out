   T  h ( intx ) {return newT(x);}
